src/components/Chat/ChatDetails.styl

```diff
@@ -32,7 +32,7 @@
         justify-content: space-between;
 
         > button {
-            min-width: 98%;
+            min-width: 100%;
```

Comment:
@BHydden   It'd be fascinating to know why you set this to 98%.    When I see a number that specific, I find myself wondering "what was special about the missing 2%, is it OK to change this?"

Reply:
I have no conscious recollection of that commit or indeed ever working on that component... 🤔

At a guess I set it to 98 because the previous setting was 48 and so I just changed the leading character? 🤷‍♂️

Reply:
Hah I can imagine that too :)

Reply:
Hey folks, i hope the issue was solved, i think online-go website should be recreated, there are too many bugs in the website. Or maybe we should consider about recreating this online-go with flutter? Flutter is really cool technology.